exec/java-exec/src/test/java/org/apache/drill/TestFunctionsWithTypeExpoQueries.java

```diff
@@ -213,7 +221,7 @@ public void tesIsNull() throws Exception {
     List<Pair<SchemaPath, TypeProtos.MajorType>> expectedSchema = Lists.newArrayList();
     TypeProtos.MajorType majorType = TypeProtos.MajorType.newBuilder()
         .setMinorType(TypeProtos.MinorType.BIT)
-        .setMode(TypeProtos.DataMode.REQUIRED)
+        .setMode(TypeProtos.DataMode.OPTIONAL)
```

Comment:
In fact, I meant to let the tests in this test-suite run with Limit-0 optimization. However, that option is off by default. So I added 
https://github.com/apache/drill/pull/489/files#diff-0e0d4b8f1e54d2a7e8b6bc04d8606bf7R35

However, that exposed an issue here:
https://issues.apache.org/jira/browse/DRILL-4656

So I think we should firstly turn this test off until we fix DRILL-4656